### DISCUSSION TITLE
Batch Processing

### DISCUSSION THREAD

**Discussion by loibi** [yermak/AudioBookConverter#309] [Q&A] [answered]
Hi,
First of all thank you for the great Software.
I am in Process converting my whole Audiobook Library from mp3 to m4b.
Basically i have 1 Folder (for Instance Asterix) with all the Albums in seperated Folders under it.
Now I have to Import 1 Album Folder, click Start, select the same Folder as Output. Then the same for the next one.

Is there any Option to make this easier?

**Comment by yermak** [comment, marked as answer]
@loibi ,

Thanks for sharing idea. I had this on my todo list for a long time, finally created issue for this. #255

Typical use case how I see it, please share your thoughs.

Have a series of books where every book in separate folder (e.g. with mp3s), ideally you would like to import only top folder and all nested books are processed separately, as a result we could have set of m4b files with 

There is workaround to achieve it now:
Add parent folder to AudobookConverter - all nested files will be added.
Switch to chapters mode.
Edit chapter (apply to all) to include file name from mp3 - this will allow to distinguish end and start of the book.
Right click on 1 chapter in next book and select split new part. This will create new book part.
Repeat this for all books.
When you click start it will create separate m4b files for every part you selected.
Unfortunately meta information will be same for all parts. (As parts normally used to limit size not for structural difference). 
But for me personally when I listen to series - details of each book in series are not that important as it's just a series...

Hope it helps.
